src/markdown-pages/install-with-kurl/setup-tls-certs.md

```diff
@@ -43,3 +43,24 @@ After the pod has been restarted direct your browser to `http://<ip>:8800` and r
     
 It's best to complete this process as soon as possible to avoid anyone from nefariously uploading TLS certificates.  After this process has completed, the vulnerability will be closed, and uploading new TLS certificates will be disallowed again.  In order to upload new TLS certificates you must repeat the steps above. 
 <br><br><br>
+
+## Registry
+
+The TLS certificate for the [registry add-on](/docs/add-ons/registry) will be renewed automatically at thirty days prior to expiration if the [ekco add-on](/docs/add-ons/ekco) is enabled with version 0.5.0+.
+
+To manually renew the certificate for the registry prior to automatic renewal, re-run the kURL install script.
```

Comment:
Will every rerun of kURL install script renew the registry certs? That's different behavior than for control plane certs, right?

Reply:
Yes to both

Reply:
Is there downtime?

Reply:
Yes, the registry pods are deleted so they will pick up the new cert.

Reply:
I guess we're OK with that because the system is more resilient to registry outages?

Reply:
It's less likely for an app to be using the registry API than the Kubernetes API but there are apps that would, e.g. kots.